httpie/downloads.py

```diff
@@ -131,7 +128,7 @@ def filename_from_url(url: str, content_type: Optional[str]) -> str:
         else:
             ext = mimetypes.guess_extension(content_type)
 
-        if ext == '.htm':  # Python 3
```

Comment:
I am not sure about that comment: does it mean you wanted to keep track for later removal of the whole code block?

Reply:
I was to note that only Python 3, but Python not 2, seems to return `.htm`.